xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -364,283 +362,189 @@ - (IBAction)IRRemoteRightArrowPressed:(UIGestureRecognizer*)sender
-    double speed = 240.0;
-    double minVelocity = 1300.0;
-    switch (sender.state)
+    case UIGestureRecognizerStateBegan:
     {
-
-      case UIGestureRecognizerStateBegan:
+      m_lastGesturePoint = translation;
+      break;
+    }
+    case UIGestureRecognizerStateChanged:
+    {
+      int keyId = 0;
+      switch (direction)
       {
-
-        if (direction == UIPanGestureRecognizerDirectionUndefined)
+        case UIPanGestureRecognizerDirectionUp:
         {
-          m_lastGesturePoint = [sender translationInView:sender.view];
-          m_lastGesturePoint.x = m_lastGesturePoint.x / 1.92;
-          m_lastGesturePoint.y = m_lastGesturePoint.y / 1.08;
-
-          m_direction = [self getPanDirection:m_lastGesturePoint];
-          m_directionOverride = false;
-        }
-        break;
-      }
-      case UIGestureRecognizerStateChanged:
-      {
-        CGPoint gesturePoint = [sender translationInView:sender.view];
-        gesturePoint.x = gesturePoint.x / 1.92;
-        gesturePoint.y = gesturePoint.y / 1.08;
-
-        CGPoint gestureMovement;
-        gestureMovement.x = gesturePoint.x - m_lastGesturePoint.x;
-        gestureMovement.y = gesturePoint.y - m_lastGesturePoint.y;
-        direction = [self getPanDirection:gestureMovement];
-
-        CGPoint velocity = [sender velocityInView:sender.view];
-        CGFloat velocityX = (0.2 * velocity.x);
-        CGFloat velocityY = (0.2 * velocity.y);
-
-        if (ABS(velocityY) > minVelocity || ABS(velocityX) > minVelocity || m_directionOverride)
-        {
-          direction = m_direction;
-          // Override direction to correct swipe errors
-          m_directionOverride = true;
-        }
-        switch (direction)
-        {
-          case UIPanGestureRecognizerDirectionUp:
-          {
-            if ((ABS(m_lastGesturePoint.y - gesturePoint.y) > speed) ||
-                ABS(velocityY) > minVelocity)
-            {
-              [g_xbmcController.inputHandler sendButtonPressed:8];
-              if (ABS(velocityY) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:8];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
-          }
-          case UIPanGestureRecognizerDirectionDown:
+          if (fabs(m_lastGesturePoint.y - translation.y) >
+              g_xbmcController.inputHandler.inputSettings.siriRemoteVerticalSensitivity)
           {
-            if ((ABS(m_lastGesturePoint.y - gesturePoint.y) > speed) ||
-                ABS(velocityY) > minVelocity)
-            {
-              [g_xbmcController.inputHandler sendButtonPressed:9];
-              if (ABS(velocityY) > minVelocity && [self shouldFastScroll])
-                [g_xbmcController.inputHandler sendButtonPressed:9];
-
-              m_lastGesturePoint = gesturePoint;
-            }
-            break;
+            CLog::Log(LOGDEBUG, "Input: Siri remote pan up (id: 23) %i",
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)